app/views/adoption_applications/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1> Adoption Application </h1>
 
 <% if @adoption_application.errors.any? %>
-  <%= render "error_messages" %>
+  <%= render 'error_messages/error_messages.html.erb', object: @adoption_application %>
```

Comment:
You can leave off the `.html.erb`. I'd also use double quotes because it's consistent with what you're doing elsewhere.